modules/ossm-control-plane-cli.adoc

```diff
@@ -12,16 +12,15 @@ You can deploy a basic `ServiceMeshControlPlane` from the command line.
 
 * The {SMProductName} Operator must be installed.
 * Access to the OpenShift CLI (`oc`).
+ifndef::openshift-rosa,openshift-dedicated[]
+* You are logged in to {product-title} as`cluster-admin`.
+endif::openshift-rosa,openshift-dedicated[]
+ifdef::openshift-rosa,openshift-dedicated[]
+* You are logged in to {product-title} as a user with the `dedicated-admin` role.
+endif::openshift-rosa,openshift-dedicated[]
 
 .Procedure
 
-. Log in to the {product-title} CLI as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
-+
-[source,terminal]
-----
-$ oc login --username=<NAMEOFUSER> https://<HOSTNAME>:6443
-----
-+
```

Comment:
**PEER REVIEWER** I removed the log in step and made it a prereq to align with other modules in the book.